src/tests/JIT/HardwareIntrinsics/Arm/Shared/_SveMinimalUnaryOpTestTemplate.template

```diff
@@ -0,0 +1,302 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
```

Comment:
This file is a copy of _SveUnaryOpTestTemplate.template with conditional tests removed as they can't be used for reduction.

Reply:
We can do `cndSel(mask, AddAcross(a), falseVal)`. The result of this would be:
- If 0th lane was active in mask, it would have the result of AddAcross(a)
- For all the other lanes, it will either have 0 (for active lanes) or falseVal for in-active lanes

If that makes sense, can you add those in your new template? This might not be super meaningful but want to make sure that we test the jit code paths at least. 
cc: @tannergooding